gluoncv/model_zoo/fcn.py

```diff
@@ -95,32 +98,36 @@ def get_fcn(dataset='pascal_voc', backbone='resnet50', pretrained=False,
     dataset : str, default pascal_voc
         The dataset that model pretrained on. (pascal_voc, ade20k)
     pretrained : bool, default False
-        Whether to load the pretrained weights for model.
+        Whether to load the pretrained weights for model. This will load FCN pretrained.
     ctx : Context, default CPU
         The context in which to load the pretrained weights.
     root : str, default '~/.mxnet/models'
         Location for keeping the model parameters.
+    backbone_pretrained : bool, default True
+        This will load pretrained backbone network, that was trained on ImageNet.
 
     Examples
     --------
     >>> model = get_fcn(dataset='pascal_voc', backbone='resnet50', pretrained=False)
     >>> print(model)
     """
+    if pretrained is True and backbone_pretrained is not True:
+        import warnings
+        warnings.warn('Ignoring backbone_pretrained and setting it to True.')
```

Comment:
forget setting pretrained_base to False

Reply:
This part is unnecessary, if FCN is pretrained, all the weights have been covered.

Reply:
if partially enabled, it will download the backbone weights first, which is not necessary

Reply:
So, the FCN pretrained weights already include the imagenet weights. Is that interpretation correct? Which implies, that I don't have to this check?  I just wanted to make sure that someone won't set FCN ``pretrained=True`` and have ``pretrained_base=False``.

Reply:
base network is a subset of whole FCN, so it pretrained=True, then it rules out everything, when pretrained is False, then pre-trained base can control whether network is partially initialized by imagenet models.

Reply:
Sure, that makes sense. I will remove this check and let the FCN's ``model.load_params`` method overload the imagenet parameters is ``pretrained=True``.